src/components/WelcomeScreen/ToolCard/ToolCard.style.ts

```diff
@@ -3,7 +3,6 @@ import { styled } from '@mui/material';
 import { alpha } from '@mui/material/styles';
 
 export const SCard = styled('div')(({ theme }) => ({
```

Comment:
Why is it called SCard? Maybe we should try to find a more self-descriptive name 😄

Reply:
Not sure if TCard does any better, but at least it´s closer to the idea of using "ToolCard". I have added that change. Open for a more descriptive name if you can think of one, though

Reply:
It can be `ToolCardContainer`. 🙂